ghost/core/test/unit/frontend/services/rendering/templates.test.js

```diff
@@ -454,28 +454,6 @@ describe('templates', function () {
             stubs.pickTemplate.calledWith('test', 'path/to/local/test.hbs').should.be.true();
         });
 
-        it('calls pickTemplate for custom routes', function () {
```

Comment:
[To help reviewers] Removed duplicated test (exactly same code as [this one](https://github.com/TryGhost/Ghost/blob/4598a342f030b2899c18fb4b699495504953c3f2/ghost/core/test/unit/frontend/services/rendering/templates.test.js#L435-L455));